pages/content/amp-dev/documentation/guides-and-tutorials/integrate/embed-stories-nonamp.md

```diff
@@ -0,0 +1,542 @@
+
+`<script async src="https://cdn.ampproject.org/amp-story-player-v0.js"></script>`
+
+`<link href="https://cdn.ampproject.org/amp-story-player-v0.css" rel="stylesheet" type="text/css">`
+
+The first script imports the logic for the AMP Story Player. The second script sets the default styling. 
+
+After including each script, include an `<amp-story-player>` element inside the document body.
+
+```html
+<amp-story-player style="width: 370px; height: 622px;">
+    <a
+      href="https://preview.amp.dev/documentation/examples/introduction/stories_in_amp/">
+      Stories in AMP - Hello World
+    </a>
+  </amp-story-player>
+```
+
+## Size the Web Story Player
+
+You may define the story player's width, height, and other styles inline or as you would any other element's style.
+
+```html
+<body>
+...
+  <amp-story-player style="width: 370px; height: 622px;">
+...
+  </amp-story-player>
+...
+</body>
+```
+
+We recommend maintaining a 69:116 aspect ratio for the best user experience, but you may define any width and height.
+
+
+### Responsive sizing
+
+The story player's responsiveness works as any other block element. Use CSS to maintain width and height ratios, such as the example below:
+html
+```
+<amp-story-player style="width: 50vw; height: 84.05vw;">
+  ...
+</amp-story-player>
+```
+
+## Configure the Web Story Player behavior
+
+The Web Story Player allows you to specify behavior, such as autoplaying stories or circular wrapping. These are specified in the Web Story JSON behavior configuration. 
+
+```html
+<amp-story-player style="width: 370px; height: 622px;">
+<script type="application/json">
+    {
+      "behavior": {
+        "autoplay": false
+      }
+    }
+  </script>
+...
+</amp-story-player>
+```
+
+# Display a Web Story 
+
+Link to a Web Story by including an `<a>` tag with the `href` attribute pointed to the desired Web Story URL within the `<amp-story-player>` element. The `href` endpoint may be the URL of a hosted Web Story or a relative path. Place the title of the story within the `<a>` tags.
+
+[example preview="top-frame" playground="true"]
+```html
+<!DOCTYPE html>
+<html>
+  <head>
+    <script
+      async
+      src="https://cdn.ampproject.org/amp-story-player-v0.js"
+    ></script>
+    <link
+      href="https://cdn.ampproject.org/amp-story-player-v0.css"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <style>
+      header {
+        height: 8vh;
+        color: #545454;
+        background-color: #DDB556;
+        display: flex;
+        align-items: center;
+        justify-content: center;
+      }
+      amp-story-player {
+        margin: 1rem auto;
+      }
+    </style>
+  </head>
+  <body>
+    <header>
+      <h1>
+        Page Header
+      </h1>
+    </header>
+    <h1>
+      Article Title
+    </h1>
+    <amp-story-player style="width: 370px; height: 622px;">
+      <a href="https://ampfest-story-player-demo.web.app/examples/amp-story/AMPFestPlayerDemo/s3"></a>
```

Comment:
@Enriqe should we write best case examples with a placeholder poster image that can be displayed until the story is loaded?

Reply:
Yep, good catch. Added the suggestion.